### DISCUSSION TITLE
Request add new site specific CSS for https://www.pixiv.net/stacc

### DISCUSSION THREAD

**Discussion by vegetax6** [W4775/Widify#7] [Ideas]
Request add new site specific CSS for https://www.pixiv.net/stacc

**Comment by W4775** [comment]
Hey thanks for the suggestion. Took a look and played around with the styling how does this look? 
![image](https://user-images.githubusercontent.com/31900031/221718167-dffda7eb-84b4-4ad1-bd79-8bdfb35fe8fe.png)

I'm working on some updates and will push this along with the other ones.
